test/browser/spec/ol/interaction/Draw.test.js

```diff
@@ -1686,6 +1686,48 @@ describe('ol/interaction/Draw', function () {
     });
   });
 
+  describe('#setType()', function () {
+    let interaction;
+
+    beforeEach(function () {
+      interaction = new Draw({
+        type: 'LineString',
+      });
+
+      expect(interaction.getActive()).to.be(true);
+
+      map.addInteraction(interaction);
+
+      // first point
+      simulateEvent('pointermove', 10, 20);
+      simulateEvent('pointerdown', 10, 20);
+      simulateEvent('pointerup', 10, 20);
+
+      expect(interaction.sketchFeature_).not.to.be(null);
```

Comment:
There shouldn't be calls to expect in `beforeEach`. Test should happen only within `it()`.
I guess this can removed except the `addInteraction` call.